File: 01_1.cpp
#include <iostream>
#include <fstream>
#include <string>

int main()
{
    std::ifstream file;
    std::string line;

    file.open("01.txt");

    while (std::getline(file, line))
    {
        std::cout << line << std::endl;
        if (line == "\n")
        {
            std::cout << "Here" << std::endl;
        }
    }

    file.close();
}

File: 01_1.py

max_calories = 0

with open("01.txt") as f:
    lines = f.readlines()


elf = 0
for line in lines:
    if line == "\n":
        if elf > max_calories:
            max_calories = elf
        elf = 0
    else:
        elf += int(line)

print(max_calories)
